.github/workflows/main.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           cp src/provisioner_lambda/*.py artifacts/provisioner_lambda
           cp src/query_lambda/*.py artifacts/query_lambda
           cp src/resources/*.sql artifacts/resources
-          cp AmazonRootCA1.pem artifacts
+          cp AmazonRootCA1.pem artifacts/common
```

Comment:
The reason for this is not obvious from the code. I think a comment explaining what's going on would be useful.

Reply:
Good idea, added @tea-short